--- wmtimer-2.91/wmtimer/Makefile.orig	2004-01-21 23:31:30.000000000 +0100
+++ wmtimer-2.91/wmtimer/Makefile	2004-01-21 23:32:07.000000000 +0100
@@ -10,10 +10,10 @@
 
 all: wmtimer
 .c.o:
-	$(CC) -g -c -O2 -Wall $< -o $*.o $(CFLAGS)
+	$(CC) -g -c $(OPTS) $< -o $*.o $(CFLAGS)
 
 wmtimer: $(OBJS)
-	$(CC) -Wall -g -o wmtimer $^ $(LIBDIR) $(LIBS)
+	$(CC) $(OPTS) -g -o wmtimer $^ $(LIBDIR) $(LIBS)
 
 install:
 	cp wmtimer /usr/local/bin/
